e2e-test/README.md

```diff
@@ -57,46 +57,65 @@ Note: The only difference between those two commands is `npm run cypress` opens
 
 This section describes how we can run and debug the tests locally with Cypress.
 
-### Run Cypress GUI with Dockerized Application (recommended)
+### Prerequisits
 
-To start the cypress GUI to select specific tests, simply start a TruBudget setup and after that, start cypress.
+Setup a provisioned Trubudget node including the excel-export service. Check out the [Developer Setup](../docs/developer/developer-setup.md) for more details.
+The easiest way to start a provisioned Trubudget node is to start it via developer script
 
-You can run application in many ways: pull built images, each service in separated terminal or in development mode. See [Developer Setup](../docs/developer/developer-setup.md) for more details.
+```bash
+bash scripts/development/start-dev.sh --slim
```

Comment:
maybe add the --excel-export to the command, so that service starts too

Reply:
Unfortunatley there is no option to enable the excel-export servcie in the dev script.
Only the operation script supports this but there I cannot rebuild the images.
I used the operation script with the --build option in the docker-compose command (modified the script)
maybe we should adapt the operation script so it's supporting image rebuild

Reply:
but i don't want the storage service to be enabled

Reply:
for now I will change it to --full - in the future if the dev script supports en/disabling services via command this can be changed again